docs/releases/pending/4411.fmf

```diff
@@ -0,0 +1,3 @@
+description: |
+  The ``bootc`` package manager now validates bootc status JSON structure
+  using Pydantic models derived from the official schema (https://bootc-dev.github.io/bootc/host-v1.schema.json).
```

Comment:
A bit misleading because we do not do the fill schema validation. This should not have any user-facing change would it?